### ISSUE TITLE
FEATURE REQUEST: Allow overriding captcha type in htmlScriptTagJsApi tag

### ISSUE THREAD

**Issue by adrienne** [biscolab/laravel-recaptcha#48]
Currently we have to set the `size` for v2 checkbox to `'normal'` or `'compact'` in the config, and it applies everywhere. However, for my use case I would like to be able to override this on certain pages. You already allow passing configuration in the JS tag, like this:
```
{!! htmlScriptTagJsApi(['lang' => 'de']) !!}
```
Can you make it so 
```
{!! htmlFormSnippet() !!}
```
can also accept configuration, at least for `size`?

**Comment by biscolab** [comment]
Hi @adrienne and thanks for your request. I'm quite busy these days, but, I should be able to do it either on Friday or next week.

**Comment by biscolab** [comment]
Hi @adrienne. FEATURE REQUEST closed by version 4.4.0. [https://github.com/biscolab/laravel-recaptcha/releases/tag/v4.4.0](https://github.com/biscolab/laravel-recaptcha/releases/tag/v4.4.0)
